### ISSUE TITLE
SuiteP - Inbound Emails - 3 suggestions

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2596]
#### Issue

1. [SUGGESTION] We need new column to which user is this inbound email assigned

2. [SUGGESTION] Now i can't sort by asc/desc the records (i have 35 inbound emails)

3. [SUGGESTION] Now i can't search the name of inboud email record - i don't see there any search field

#### Steps to Reproduce

1. Go to Admin
2. Clik on Inbouds Emails

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [SuiteP - Inbound Emails - 3 suggestions](https://trello.com/c/wdaW5Cjd/53-suitep-inbound-emails-3-suggestions)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
